src/argilla/labeling/text_classification/rule.py

```diff
@@ -74,6 +74,31 @@ def author(self):
         """Who authored the rule."""
         return self._author
 
+    def update_label(self, label):
+        """Updates the label"""
+        self._label = label
+
+    def add_to_dataset(self, dataset: str):
+        """Add to rule to the given dataset"""
+        labeling_rule = LabelingRule(
+            query=self.query, label=self.label, labels=[], author=self.author or "None"
+        )
+        api.active_api().add_dataset_labeling_rules(dataset, rules=[labeling_rule])
+
+    def remove_from_dataset(self, dataset: str):
+        """Removes the rule from the given dataset"""
+        labeling_rule = LabelingRule(
+            query=self.query, label=self.label, labels=[], author=self.author or "None"
+        )
+        api.active_api().delete_dataset_labeling_rules(dataset, rules=[labeling_rule])
+
+    def update_at_dataset(self, dataset: str):
+        """Updates the rule at the given dataset"""
+        labeling_rule = LabelingRule(
+            query=self.query, label=self.label, labels=[], author=self.author or "None"
```

Comment:
Does work for multi label rules? I think we can have errors since rule.label can be a list